test/gtest/ucp/test_ucp_am.cc

```diff
@@ -772,19 +772,38 @@ class test_ucp_am_nbx_dts : public test_ucp_am_nbx {
                                        UCS_BIT(UCP_DATATYPE_IOV) |
                                        UCS_BIT(UCP_DATATYPE_GENERIC);
 
+    virtual ucp_ep_params_t get_ep_params()
+    {
+        ucp_ep_params_t ep_params = test_ucp_am_nbx::get_ep_params();
+
+        if (get_variant_value(2) == UCP_EP_PARAM_FIELD_ERR_HANDLING_MODE) {
+            ep_params.field_mask |= UCP_EP_PARAM_FIELD_ERR_HANDLING_MODE;
+            ep_params.err_mode    = UCP_ERR_HANDLING_MODE_PEER;
+        }
+
+        return ep_params;
+    }
+
     static void get_test_dts(std::vector<ucp_test_variant>& variants)
     {
         /* coverity[overrun-buffer-val] */
         add_variant_values(variants, test_ucp_am_base::get_test_variants,
                            dts_bitmap, ucp_datatype_class_names);
     }
 
-    static void get_test_variants(std::vector<ucp_test_variant>& variants)
+    static void get_test_dts_reply(std::vector<ucp_test_variant>& variants)
     {
         add_variant_values(variants, get_test_dts, 0);
         add_variant_values(variants, get_test_dts, UCP_AM_SEND_REPLY, "reply");
     }
 
+    static void get_test_variants(std::vector<ucp_test_variant>& variants)
+    {
+        add_variant_values(variants, get_test_dts_reply, 0);
+        add_variant_values(variants, get_test_dts_reply,
+                           UCP_EP_PARAM_FIELD_ERR_HANDLING_MODE, "errh");
```

Comment:
can we pass UCP_ERR_HANDLING_MODE_PEER / UCP_ERR_HANDLING_MODE_NONE as the variant values?